docker_images/pythonv2/wrapper/python_glue/internal_wrapper_glue.py

```diff
@@ -39,5 +42,9 @@ def get_capabilities():
             "dropped_connection_tests": True,
             "net_control_app": True,
         },
-        "skip_list": ["invokesModuleMethodCalls", "invokesDeviceMethodCalls"],
+        "skip_list": []
     }
+    if not getattr(IoTHubModuleClient, "invoke_method", None):
+        caps["skip_list"].append('invokesDeviceMethod')
```

Comment:
>invokesDeviceMethod [](start = 34, length = 19)

I should have told you to never trust my code, especially when I send it in teams. Should be `invokesDeviceMethodCalls` and `invokesModuleMethodCalls`.